app/questionnaire/questionnaire_schema.py

```diff
@@ -1385,27 +1385,48 @@ def get_item_anchor(self, section_id: str, list_name: str) -> str | None:
                 if item["for_list"] == list_name and item.get("item_anchor_answer_id"):
                     return f"#{str(item['item_anchor_answer_id'])}"
 
+    def _update_placeholder_dependencies(
+        self, placeholder_answer_ids: set, block: ImmutableDict
```

Comment:
Type hings for answer ids needs to be stricter. Also, you don't need `placeholder` in the variable name, it is already in the context of placeholders.

Reply:
Updated 👍